.gitignore

```diff
@@ -36,3 +36,4 @@ zombienet-linux
 # IDEs
 .vscode/
 .idea/
+polkadot-sdk
```

Comment:
Guess it'd be cleaner not to mix repositories in first place. In this case, moving your copy of `polkadot-sdk` out of `virto-node`'s repo seems cleaner than ignoring it.